usaspending_api/awards/models_matviews/vw_search_award.py

```diff
@@ -0,0 +1,88 @@
+from django.contrib.postgres.fields import ArrayField
+from django.contrib.postgres.search import SearchVectorField
+from django.db import models
+
+from usaspending_api.awards.models import Award
+
+
+class ViewSearchAward(models.Model):
```

Comment:
For consideration - in the past working with ORMs, I a sweetspot in naming conventions seemed to be suffixing the classes with *what* they were (nothing for tables, `*View` for views, `*MatView` for materialized views, etc.)  ... but _prefixing_ database objects with what they were (e.g. `vw_*` or `mv_*` or `ix_*` ... ). 

I think it was just a subtle thing that
- when working through the code, you're trying to find that functional class, the thing that serves some business function (business model focused) 
- but when browsing through the db or writing sql, you often wanted to go to a type of object (get to the views, get to the matview, etc.). (physical model focused)

I'm fine either way though as long as we get to a consistent point.